maintenance_equipment_tags/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
```

Comment:
You are missing something here. The group base.group_user has only red access, which is fine. But nobody has rights to modify, create and delete tags. I think any maintenance manager should have such rights.

Reply:
Added rule: `access_maintenance_equipment_man,access_maintenance_equipment_tag,model_maintenance_equipment_tag,maintenance.group_equipment_manager,1,1,1,1`